### PR TITLE
BL-1: Add named reactions

### LINKED ISSUE #67: AF-1: Add the Named Reactions in Organic Synthesis
Named Reactions List

### PR DESCRIPTION
**Pull request by Sulstice**

### CODE REVIEW

**Comment by Sulstice** [comment]
I am working on this now. Time start.

**Comment by Sulstice** [comment]
@LierB 

```python
        'pinner_reaction': 'CC#N.CC(OC)=N',
```

is this correct?

Screen Shot 2022-06-04 at 7 00 27 PM

**Comment by LierB** [comment]
@Sulstice 

if I would have written it in the ionic form
```
'pinner_reaction': 'CC#N.CC(OC)=[NH2+]',
```
instead of 
```
'pinner_reaction': 'CC#N.CC(OC)=N',
```
it should be correct as it is a medium product, the pinner salt.
the final product depends on the reactants:
- water: ester
- excess of alcohol: orthoester
- ammonia or amine: amidine
- di-nitrils: imidine
- hydrogen sulfide: thionoester

**Comment by Sulstice** [comment]
Dont remove it. Stupid machine, I'm working on it.

**Comment by Sulstice** [comment]
Don't remove it , it's a work in progress

File: global_chem/global_chem/organic_synthesis/named_reactions_in_organic_synthesis/named_reactions_in_organic_synthesis.py
Comment (ANUGAMAGE) [inline global_chem/global_chem/organic_synthesis/named_reactions_in_organic_synthesis/named_reactions_in_organic_synthesis.py:1]: Conflicts were resolved
Sul if you can please review this code. It's quite old. So it is better to recheck before merging in.

**Review by ANUGAMAGE — COMMENTED** [review]
Successfully Merged

**Pull request merged**